src/wrapper/objects/jvalue.rs

```diff
@@ -175,10 +175,16 @@ impl<'a> From<JObject<'a>> for JValue<'a> {
     }
 }
 
-// jbool
 impl<'a> From<bool> for JValue<'a> {
     fn from(other: bool) -> Self {
-        JValue::Bool(other as jboolean)
+        JValue::Bool(if other { JNI_TRUE } else { JNI_FALSE })
```

Comment:
I think this is needed. Otherwise the user needs to add this `if other { JNI_TRUE } else { JNI_FALSE }` in their code, when they want to create a `JValue` from a `bool`. The other types are okay, as you can use them directly, but for `bool` you mostly use `bool` and don't want to use `jboolean`.